chain-impl-mockchain/src/transaction/mod.rs

```diff
@@ -1,7 +1,7 @@
-mod transaction;
-mod transfer;
-mod utxo;
-mod witness;
+pub mod transaction;
+pub mod transfer;
+pub mod utxo;
+pub mod witness;
```

Comment:
We should not need to make this interface public, allthe APIs are re-exported below with `pub use self::transaction::*;` and following lines.

Reply:
Done